### PR TITLE
Pbi/25844 24538 add documentation for is permitted everywhere

### PR DESCRIPTION
**Pull request by CPatPundZ**
Added is-permitted-everywhere.md with some information about this feature

### CODE REVIEW

File: docs/admin/services/intrafind-indexer/installation.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 - OS-Version: Windows Server 2012 or higher
 - IIS-Version: 7.5 or higher
 - SQL-Server-Version: 2016 or higher
-- .NET Core Version: 2.1.4 or higher
-- .NET Core Windows Server Hosting-Version: 2.1.4 or higher
+- .NET Core Version: 3.1 or higher
+- .NET Core Windows Server Hosting-Version: 3.1 or higher
```
Comment (thomashauser) [inline docs/admin/services/intrafind-indexer/installation.md]: We tested only 3.1!

File: docs/admin/services/intrafind-indexer/installation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 ### Setting up a new site on IIS
 
-1. Download and install [.NET Core Windows Server Hosting](https://www.microsoft.com/net/download/dotnet-core/2.1)  (2.1.4 or later)
+1. Download and install [.NET Core Windows Server Hosting](https://download.visualstudio.microsoft.com/download/pr/c9206d6d-a11a-4b0b-834b-6320c44d0a2d/993571f75a96b6a64f8bca001797c4f0/dotnet-hosting-3.1.1-win.exe)  (3.1.1 or later)
```
Comment (thomashauser) [inline docs/admin/services/intrafind-indexer/installation.md]: We tested only 3.1.1!

File: docs/admin/services/intrafind-indexer/installation.md [hunk 3]
```diff
@@ -36,7 +36,7 @@
 ### Preparations
 
 - Ensure Powershell Version 5.0 or higher is available. Download [WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=50395)
-- Download and install [.NET Core Windows Server Hosting Package](https://www.microsoft.com/net/download/dotnet-core/2.1) (2.1.4 or later)
+- Download and install [.NET Core Windows Server Hosting Package](https://download.visualstudio.microsoft.com/download/pr/c9206d6d-a11a-4b0b-834b-6320c44d0a2d/993571f75a96b6a64f8bca001797c4f0/dotnet-hosting-3.1.1-win.exe) (3.1.1 or later)
```
Comment (thomashauser) [inline docs/admin/services/intrafind-indexer/installation.md]: We tested only 3.1.1!

File: docs/admin/services/intrafind-search-proxy/installation.md [hunk 4]
```diff
@@ -18,7 +18,7 @@
 
 ### Setting up a new site on IIS
 
-1. Download and install [.NET Core Windows Server Hosting](https://www.microsoft.com/net/download/dotnet-core/2.1)  (2.1.4 or later)
+1. Download and install [.NET Core Windows Server Hosting](https://download.visualstudio.microsoft.com/download/pr/c9206d6d-a11a-4b0b-834b-6320c44d0a2d/993571f75a96b6a64f8bca001797c4f0/dotnet-hosting-3.1.1-win.exe)  (3.1.1 or later)
```
Comment (thomashauser) [inline docs/admin/services/intrafind-search-proxy/installation.md]: We tested only 3.1.1!

File: docs/admin/services/intrafind-search-proxy/installation.md [hunk 5]
```diff
@@ -5,8 +5,8 @@
 - OS-Version: Windows Server 2012 or higher
 - IIS-Version: 7.5 or higher
 - SQL-Server-Version: 2016 or higher
-- .NET Core Version: 2.1.4 or higher
-- .NET Core Windows Server Hosting-Version: 2.1.4 or higher
+- .NET Core Version: 3.1 or higher
+- .NET Core Windows Server Hosting-Version: 3.1 or higher
```
Comment (thomashauser) [inline docs/admin/services/intrafind-search-proxy/installation.md]: We tested only 3.1!

File: docs/admin/services/intrafind-search-proxy/installation.md [hunk 6]
```diff
@@ -36,7 +36,7 @@
 ### Preparations
 
 - Ensure Powershell Version 5.0 or higher is available. Download [WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=50395)
-- Download and install [.NET Core Windows Server Hosting Package](https://www.microsoft.com/net/download/dotnet-core/2.1) (2.1.4 or later)
+- Download and install [.NET Core Windows Server Hosting Package](https://download.visualstudio.microsoft.com/download/pr/c9206d6d-a11a-4b0b-834b-6320c44d0a2d/993571f75a96b6a64f8bca001797c4f0/dotnet-hosting-3.1.1-win.exe) (3.1.1 or later)
```
Comment (thomashauser) [inline docs/admin/services/intrafind-search-proxy/installation.md]: We tested only 3.1.1!

File: docs/admin/services/intrafind-indexer/consoleapplication.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    [Error] "Event HTTP request call: The remote server returned an error: (502) Bad Gateway. for URL 'https://localhost:44316/'" [...]
+
+The given URL has to be replaced with the microservice URL, of course. You can filter for this microservice URL to find out errors regarding the microservice in Symbio's logs.
+
+### Sample fullfetch log entries
+
+<pre><code>
+2019-02-18 15:53:53.808 +01:00 [INF] Crawling started for tenant database '21cea141-b334-4c72-9b0c-bb84b5465a7b'...
+2019-02-18 15:53:54.847 +01:00 [INF] #Documents known to Intrafind: 338
+2019-02-18 15:53:58.847 +01:00 [INF] 		 Found valid/released 'mainProcess', version 2.0, id '0d12c107-5b73-4afa-9636-cbc1a43f505e' and versionId '5a1db2fc-11ae-4f00-9cb5-5899fc747167'
```
Comment (thomashauser) [inline docs/admin/services/intrafind-indexer/consoleapplication.md]: The storage id and element ids are ids from a test run and not from a prod db?

Comment (CPatPundZ) [reply docs/admin/services/intrafind-indexer/consoleapplication.md]: yes, a test run, 338 documents seems to be showcase storage

File: docs/admin/services/intrafind-search-proxy/installation.md (hunk 6 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-search-proxy/installation.md]: solved

File: docs/admin/services/intrafind-search-proxy/installation.md (hunk 5 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-search-proxy/installation.md]: solved

File: docs/admin/services/intrafind-search-proxy/installation.md (hunk 4 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-search-proxy/installation.md]: solved

File: docs/admin/services/intrafind-indexer/installation.md (hunk 3 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-indexer/installation.md]: solved

File: docs/admin/services/intrafind-indexer/installation.md (hunk 2 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-indexer/installation.md]: solved

File: docs/admin/services/intrafind-indexer/installation.md (hunk 1 above)
Comment (CPatPundZ) [reply docs/admin/services/intrafind-indexer/installation.md]: solved

**Pull request merged**